### PR TITLE
Changes for WordPress 5.7

### PR DESCRIPTION
**Pull request by danielpwaldschmidt**
…label prop, and use ToolbarButton components for choosing which opportunity type to show.

### CODE REVIEW

File: admin/class-admin.php [hunk 1]
```diff
@@ -677,11 +677,19 @@ public function display_opportunity_email_form_meta_box( $opp ){
 			$editor_id      = 'volunteer-email-editor';
 			$content        = get_option( $editor_id );
 
-			// Set the editor options array
+			/**
+			 * Set the editor options array.
+			 *
+			 * Due to a bug in WordPress 5.6+, the wp_editor instance has to be
+			 * output with the 'Text' tab selected by default.
+			 *
```
Comment (jg314) [inline admin/class-admin.php:685]: What do you think about linking to that trac ticket here? I could see that coming in handy the next time we look over this plugin.

File: README.txt [hunk 2]
```diff
@@ -109,6 +109,11 @@ If the recurring opportunity will have the same volunteers each time, we’d rec
 
 == Changelog ==
 
+= 1.4.5 =
+* Updated the Toolbar component of the Volunteer Opportunities block.
```
Comment (jg314) [inline README.txt]: Can you add a "why" here? A little more detail will probably be helpful down the road.

File: admin/js/wi-volunteer-management-block.js [hunk 3]
```diff
@@ -3,7 +3,7 @@
 	const { registerBlockType }                  = wp.blocks;
 	const { createElement, Fragment }            = wp.element;
 	const { BlockControls }                      = wp.blockEditor;
-	const { Disabled, Toolbar }                  = wp.components;
+	const { Disabled, Toolbar, ToolbarButton }                  = wp.components;
```
Comment (jg314) [inline admin/js/wi-volunteer-management-block.js]: Can we remove the extra spaces on this line to make `= wp.components;` line up again?

File: admin/js/wi-volunteer-management-block.js [hunk 4]
```diff
@@ -91,22 +91,20 @@
...
-								},
-								{
-									icon: flexibleOppsIcon,
-									title: __( 'Show Flexible Opportunities', 'wired-impact-volunteer-management' ),
-									onClick: () => setAttributes( { showOneTime: false } ),
-									isActive: showOneTime !== true,
-								},
-							]
-						} />
+						<Toolbar label="Opportunity Type Controls">
```
Comment (jg314) [inline admin/js/wi-volunteer-management-block.js]: Can we wrap the label in the `__()` translation function like the other text here?

File: admin/js/admin.js [hunk 5]
```diff
@@ -283,6 +283,17 @@
...
         if ( $( email_editor ).exists() ) {
 
+			/**
+			 * Due to a bug in WordPress 5.6+, the wp_editor instance has to be
+			 * output with the 'Text' tab selected by default.
+			 *
+			 * This code clicks the 'Visual' tab to switch to the TinyMCE editor,
+			 * which is what most users will want to use.
+			 */
+			setTimeout( function() {
```
Comment (jg314) [inline admin/js/admin.js:295]: Nice job getting a fix in place here. Not ideal give the bug, but you found a way!

File: admin/js/admin.js [hunk 6]
```diff
@@ -283,6 +283,17 @@
 
         if ( $( email_editor ).exists() ) {
 
+			/**
+			 * Due to a bug in WordPress 5.6+, the wp_editor instance has to be
```
Comment (jg314) [inline admin/js/admin.js:287]: Same note here about that trac ticket. I could see it being helpful to list it here as well.

File: README.txt [hunk 7]
```diff
@@ -4,7 +4,7 @@ Tags: nonprofits, non profits, not-for-profit, volunteers, volunteer
 Requires at least: 4.0
 Tested up to: 5.6
 Requires PHP: 5.2.4
-Stable tag: 1.4.4
+Stable tag: 1.4.5
```
Comment (jg314) [inline README.txt:7]: Nice work on all of this sir!

**Review by jg314 — CHANGES_REQUESTED** [review]

File: admin/class-admin.php (hunk 1 above)
Comment (danielpwaldschmidt) [reply admin/class-admin.php:685]: Added.

File: admin/js/wi-volunteer-management-block.js (hunk 3 above)
Comment (danielpwaldschmidt) [reply admin/js/wi-volunteer-management-block.js]: Done.

File: admin/js/wi-volunteer-management-block.js (hunk 4 above)
Comment (danielpwaldschmidt) [reply admin/js/wi-volunteer-management-block.js]: Done.

File: admin/js/admin.js (hunk 6 above)
Comment (danielpwaldschmidt) [reply admin/js/admin.js:287]: Done.

File: README.txt (hunk 2 above)
Comment (danielpwaldschmidt) [reply README.txt]: Done.

**Pull request merged**